mandatory/4-tax.js

```diff
@@ -17,7 +19,9 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(sales) {
+  return "£" + calculateSalesTax(sales).toFixed(2);
```

Comment:
Hi @momahboobian 
I think better practice is to use
`£${result}` - backquote symbols and $ in your return

Reply:
Sure, Thank you 😊